packages/next/src/client/components/react-dev-overlay/internal/container/RuntimeError/GroupedStackFrames.tsx

```diff
@@ -5,62 +5,56 @@ import { FrameworkIcon } from './FrameworkIcon'
 function FrameworkGroup({
   framework,
   stackFrames,
-  all,
 }: {
   framework: NonNullable<StackFramesGroup['framework']>
   stackFrames: StackFramesGroup['stackFrames']
-  all: boolean
 }) {
   return (
-    <>
-      <details data-nextjs-collapsed-call-stack-details>
-        <summary
-          tabIndex={10} // Match CallStackFrame tabIndex
+    <details data-nextjs-collapsed-call-stack-details>
+      {/* Match CallStackFrame tabIndex */}
+      <summary tabIndex={10}>
+        <svg
+          data-nextjs-call-stack-chevron-icon
+          fill="none"
+          height="20"
+          width="20"
+          shapeRendering="geometricPrecision"
+          stroke="currentColor"
+          strokeLinecap="round"
+          strokeLinejoin="round"
+          strokeWidth="2"
+          viewBox="0 0 24 24"
         >
-          <svg
-            data-nextjs-call-stack-chevron-icon
-            fill="none"
-            height="20"
-            width="20"
-            shapeRendering="geometricPrecision"
-            stroke="currentColor"
-            strokeLinecap="round"
-            strokeLinejoin="round"
-            strokeWidth="2"
-            viewBox="0 0 24 24"
-          >
-            <path d="M9 18l6-6-6-6" />
-          </svg>
-          <FrameworkIcon framework={framework} />
-          {framework === 'react' ? 'React' : 'Next.js'}
-        </summary>
-
-        {stackFrames.map((frame, index) => (
-          <CallStackFrame key={`call-stack-${index}-${all}`} frame={frame} />
-        ))}
-      </details>
-    </>
+          <path d="M9 18l6-6-6-6" />
+        </svg>
+        <FrameworkIcon framework={framework} />
+        {framework === 'react' ? 'React' : 'Next.js'}
+      </summary>
+      {stackFrames.map((frame, index) => (
+        <CallStackFrame key={`call-stack-${index}`} frame={frame} />
+      ))}
+    </details>
   )
 }
 
 export function GroupedStackFrames({
   groupedStackFrames,
-  all,
+  show,
 }: {
   groupedStackFrames: StackFramesGroup[]
-  all: boolean
+  show: boolean
 }) {
+  if (!show) return
```

Comment:
We should respect the main "Hide/Show collapsed frames" and should not show grouped stackframes at all when collapsed